src/boilerplate/content.js

```diff
@@ -37,7 +37,7 @@ export default {
     },
 
     getGameContent(key) {
-        return this._gameContent?.[key];
+        return () => this._gameContent?.[key];
```

Comment:
why does this need to be a function?  If you're worried about the data getting dereferenced when loading through the window message event, you can use Object.assign on the this._gameContent object to replace data instead of changing the data reference through assignment.